spec/features/idv/flow_spec.rb

```diff
@@ -347,7 +347,9 @@
     end
 
     scenario 'pick USPS address verification' do
-      sign_in_and_2fa_user
+      user = sign_in_and_2fa_user
+      loa3_sp_session
```

Comment:
Agreed, but I think we only need one test that uses SAML and goes through everything as a user would.